pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -114,6 +114,8 @@ message CertManagerConfig {
   string podDnsPolicy = 10;
 
   TypeMapStringInterface podDnsConfig = 11;
+
+  TypeSliceOfMapStringInterface tolerations = 12 [deprecated=true];
```

Comment:
Can we use the k8s type here?

Reply:
Do you mean the IstioControlPlaneSpec? we use TypeSliceOfMapStringInterface for tolerations in values schema here

Reply:
We may as well use it here too. That's true for other fields that are currently untyped too. I've created https://github.com/istio/istio/issues/19709 to track this, you can skip it for this PR.